diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/ASTNodeUtilsTest.kt

```diff
@@ -62,51 +73,516 @@ class ASTNodeUtilsTest {
     }
 
     @Test
-    fun `moveChildBefore 1 - Should correctly move node child before another`() {
-        applyToCode("""
-                |val a = 0
-                |val b = 1
-            """.trimMargin()) { node ->
-            if (node.elementType == CLASS_BODY) {
-                val val1 = node.getFirstChildWithType(PROPERTY)
-                val val2 = val1!!.nextSibling { it.elementType == PROPERTY }!!
-                val whiteSpace = val1.treeNext.clone() as ASTNode
-                node.moveChildBefore(val2, val1, false)
-                node.addChild(whiteSpace, val1)
-                node.removeChild(val1.treeNext)
-                Assert.assertTrue(node.text == """
-                    |val b = 1
-                    |val a = 0
-                    |
-                    """.trimMargin()
-                )
+    fun `test node's check text lenght`(){
+        val code = """
+            class Test {
+                /**
+                * test method
+                * @param a - dummy int
+                */
+                fun foo(a: Int): Int = 2 * a
+            }
+        """.trimIndent()
+        var counter = 0
+        applyToCode(code){node ->
+            if (node.elementType == CLASS) {
+                Assert.assertTrue(node.checkLength(IntRange(code.length, code.length)))
+                counter++
             }
         }
+        Assert.assertEquals(1, counter)
     }
 
     @Test
-    fun `moveChildBefore 2 - Should correctly move node child to the end`() {
+    fun `test IdentifierName`(){
+        val code = """
+            class Test {
+                /**
+                * test method
+                * @param a - dummy int
+                */
+                fun foo(a: Int): Int = 2 * a
+            }
+        """.trimIndent()
+        var counter = 0
+        val list = listOf("Test", "foo", "a", "a", "Int", "Int", "a")
+        applyToCode(code){ node ->
+            if (node.getIdentifierName() != null) {
+                Assert.assertEquals(list[counter], node.getIdentifierName()!!.text)
+                counter++
+            }
```

Comment:
```suggestion
            node.getIdentifierName()?.let {
                Assert.assertEquals(list[counter], it.text)
                counter++
            }
```
Once again, code can be more compact and lets you avoid calling same method twice and using `!!` inside `if` block.